CMakeLists.txt

```diff
@@ -17,6 +17,8 @@ cmake_minimum_required( VERSION 3.20 FATAL_ERROR )
 # For that, we first try to find ecbuild (QUIETly) ...
 set( MINIMUM_ECBUILD_VERSION 3.8.5 )
 find_package( ecbuild ${MINIMUM_ECBUILD_VERSION} HINTS ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/../ecbuild QUIET )
+set( CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake )
+include( ecrad_macros )
```

Comment:
Can we move this include after the `project` definition on l.42? Main reason for moving this further down is that ecrad ships with an ecbuild bootstrap mechanism and it seems sensible to make sure that happens before including local macros.

To put it after `project` specifically is to ensure that these includes happen within the project scope, similarly to how ectrans does this:
https://github.com/ecmwf-ifs/ectrans/blob/46e565379a4069a722f3659920d432a5fd3fbb09/CMakeLists.txt#L14

The `CMAKE_MODULE_PATH` modification should also be unnecessary.